logging/src/main/java/org/jboss/as/logging/handlers/ConsoleHandlerResourceDefinition.java

```diff
@@ -50,7 +50,7 @@
             .setAttributeMarshaller(ElementAttributeMarshaller.NAME_ATTRIBUTE_MARSHALLER)
             .setDefaultValue(new ModelNode(Target.SYSTEM_OUT.toString()))
             .setResolver(TargetResolver.INSTANCE)
-            .setValidator(EnumValidator.create(Target.class, true, false))
```

Comment:
same remark, there was a mismatch between `setAllowExpression(true)` and the EnumValidator 2nd boolean parameter

Reply:
Hi, I think the `setAllowExpression(true)` is correct because the `validateParameter()` method in the `EnumValidator` class doesn't seem to check whether the value is true or false.

https://github.com/wildfly/wildfly-core/blob/main/controller/src/main/java/org/jboss/as/controller/operations/validation/EnumValidator.java#L220